infra/dockerfile.commit_retrieval

```diff
@@ -2,18 +2,22 @@ FROM mozilla/bugbug-base:latest
 
 COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
-# Mercurial need Python2 :(
+# Setup mercurial from its own website, without install pip2 which has a lot of dependencies
+ARG MERCURIAL_VERSION="5.1"
+
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
 RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
-    python2 -m pip install --disable-pip-version-check --no-cache-dir mercurial==5.1 && \
+    apt-get install --no-install-recommends -y python python-dev python-docutils build-essential git curl && \
```

Comment:
Why is pyhton-docutils needed? We are not installing it in https://github.com/mozilla/code-coverage/blob/master/bot/ci/bootstrap.sh.

Reply:
Using the make method required python-docutils, otherwise, the compilation fails partway and the  docker build fails.

Reply:
I've figured out how to solve the error that runnning "python /opt/mercurial-$MERCURIAL_VERSION/setup.py install" gives. So I'll be reverting back to the old method of installation and won't be needing python-docutils in the future.